.travis.yml

```diff
@@ -0,0 +1,18 @@
+# references:
+# # * http://www.objc.io/issue-6/travis-ci.html
+# # * https://github.com/supermarin/xcpretty#usage
+#
+
+language: objective-c
+osx_image: xcode8
+cache: cocoapods
+podfile: Podfile
+before_install:
+- rvm install $(cat .ruby-version) 
+- rvm use $(cat .ruby-version)
+- gem install bundler
+- bundle install
+- pod install
+script:
+- set -o pipefail && xcodebuild test -project Simcoe.xcodeproj -scheme SimcoeTests -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.0' | xcpretty
```

Comment:
Changing `-project Simcoe.xcodeproj` to `-workspace Simcoe.xcworkspace` seems to fix the issues on my local computer...